.github/workflows/lint.yml

```diff
@@ -36,14 +36,10 @@ jobs:
       with:
         files: |
           **/*.js
+          **/*.json
 
     - name: Run JS linting
-      run: |
-        if [[ "${{ steps.changed-files.outputs.any_changed }}" == 'true' && "${{ github.event_name }}" == "pull_request" ]]; then
-          npx wp-scripts lint-js ${{ steps.changed-files.outputs.all_changed_files }}
-        elif [[ "${{ github.event_name }}" == "push" ]]; then
-          npm run lint:js
-        fi
+      run: npm run lint:js
```

Comment:
So, I added `*.json` to the list in case any linting or prettier rules dependency is updated, then the action will lint using the updated rules. Due to this, `npx wp-scripts lint-js ${{ steps.changed-files.outputs.all_changed_files }}` runs on the `*.json` when it is part of the diff due to which the linting fails.